apps/spark/src/main/java/com/linkedin/openhouse/jobs/util/DatabaseTableFilter.java

```diff
@@ -24,6 +30,11 @@ public boolean applyTableName(String tableName) {
     return tablePattern.matcher(tableName).matches();
   }
 
+  public boolean applyTableCreationTime(long creationTime) {
+    long st = System.currentTimeMillis() - TimeUnit.HOURS.toMillis(minAgeThresholdHours);
```

Comment:
Took some time to understand the PR. Now it makes sense time as filtering is done as part of database table filtering.